src/contexts/plotController/plotControllerContext.tsx

```diff
@@ -46,11 +46,12 @@ const plotEventsUserContext =
 
 export function usePlotEvents(
   handlers: EventsHandlers,
-  options?: ControllerHookOptions,
+  options: ControllerHookOptions = {},
 ) {
-  const id = options?.controllerId;
-  const userContext = plotEventsUserContext.useNestedContext(id);
-  if (!userContext) {
+  const { controllerId: id, disabled = false } = options;
+
+  const userContext = !disabled && plotEventsUserContext.useNestedContext(id);
```

Comment:
It is not allowed to call a hook conditionally. We need to always call `useNestedContext`